packages/esm-appointments-app/src/appointments-calendar/daily/daily-calendar-view.component.tsx

```diff
@@ -23,7 +23,7 @@ const DailyCalendarView: React.FC<DailyCalendarViewProps> = ({
 }) => {
   return (
     <div className={styles.container}>
-      <DailyHeader type={type} currentDate={currentDate} setCurrentDate={setCurrentDate} events={events} />
+      <DailyHeader type={type} currentDate={currentDate} setCurrentDate={setCurrentDate} />
```

Comment:
there were some places where this prop wasn't actually being used, so I removed it in those cases.